api/report-templates/bcAnnualReport.html

```diff
@@ -15,7 +15,7 @@
             [[logo.html]]
           </td>
           <td>
-            <div class="report-type">{{ filing_date_time.split()[2] }} ANNUAL REPORT</div>
+            <div class="report-type">{{ annualReport.annualReportDate[:4] }} ANNUAL REPORT</div>
```

Comment:
Probably could use annualReport.annualReportDate | date:"%Y" to avoid potential issue with slicing when data type is not correct? But if it's working fine now then all good

Reply:
Okay, ill change it.

Reply:
Probably we should stick with your current implementation